inc/saiport.h

```diff
@@ -531,6 +539,18 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_ADVERTISED_MEDIA_TYPE,
 
     /**
+     * @brief Query/Configure Port's Advertised OUI code
+     *
+     * Organizationally Unique Identifier for 25G/50G auto negotiation.
+     * Default is 0x6A737D for 25G Ethernet Consortium
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0x6A737D
```

Comment:
@kcudnik - 
i already added support for hex numbers, just need to be checked in #432 
This is pushed to master, my pull request to 0.9.6. Can you push #432 to 0.9.6 also.